### DISCUSSION TITLE
Images won't load

### DISCUSSION THREAD

**Discussion by Treekaya** [requarks/wiki#7490] [Error / Bug Report]
I am self hosting wiki.js with version: 2.5.305
It is being hosted via docker.
wiki: ghcr.io/requarks/wiki:2 
wiki-update-companion: ghcr.io/requarks/wiki-update-companion:latest
db: postgres:11 

Recently no images are loading in. The Browser returns a 500 Internal Server Error. I have traced it back to this log:
2024-12-23T08:37:56.272Z [MASTER] error: path must be absolute or specify root to res.sendFile

Looking online I also found similar problems:
https://github.com/Requarks/wiki/issues/2930 (Closed Jan 2021)
https://github.com/requarks/wiki/issues/3041 (Closed Feb 2021)
https://github.com/requarks/wiki/discussions/5223 (Unanswered Apr 2022)

How to reproduce:
Create or edit a page and upload via the Wiki.js UI an Image.

They usually have a solution where they are changing the code but since I am working with the docker container that, I was unable to try it.
I have no idea why it happened now. The Wiki got last updated around a month, so it didn't even show up straight after the last update or anything, I am guessing something else is causing this.

**Comment by lostingu** [comment]
我也碰到了该问题。
NAS上docker部署，编辑md格式，上传图片后，过一段时间（可能几天可能几周），打开后无法加载图片。必须要删除图片重新上传即可，过一段时间又无法加载图片。

根据备份、导出内容，看到上传的图片文件一直都在，md页面插入图片管理中，也显示有该图片文件。但一段时间后总是无法显示图片了。
![image](https://github.com/user-attachments/assets/de866feb-fe62-42a8-8f5e-cb76af849666)


希望尽快支持“上传文件路径管理”功能，感觉还是上传文件的保存路径有点问题。

**Comment by jcdufourd** [comment]
I had the same problem of images not appearing, and solved it by changing the "Path" of the "Local File System" in the "Storage" section of the admin area from **_relative_** to **_absolute_** (as required in the comment below the field)
